mod/ckeditor/views/default/ckeditor/init.php

```diff
@@ -7,12 +7,14 @@
 
 ?>
 <script>
-// This global variable must be set before the editor script loading.
-CKEDITOR_BASEPATH = elgg.config.wwwroot + 'mod/ckeditor/vendors/ckeditor/';
+require(['elgg'], function (elgg) {
+	// This global variable must be set before the editor script loading.
```

Comment:
Oops, anything AMD will be long-cached. Really we need `elgg.fetchConfig('myplugin')` which would return a Promise. This frees us to provide this data any way we want.